update-stable-ids/src/main/java/org/reactome/release/updateStableIds/Main.java

```diff
@@ -27,33 +27,34 @@ public static void main( String[] args ) throws Exception
     		pathToConfig = "src/main/resources/config.properties";
     	}
        
+	System.out.println(pathToConfig);
        //Sets up the various DB Adaptors needed. This includes the current and previous test_slice versions on the release server, 
        //as well as gk_central on the curation server.
        Properties props = new Properties();
        props.load(new FileInputStream(pathToConfig));
-       int port = Integer.valueOf(props.getProperty("port"));
+       int port = Integer.valueOf(props.getProperty("release.database.port"));
        int releaseNumber = Integer.valueOf(props.getProperty("releaseNumber"));
        int prevReleaseNumber = releaseNumber - 1;
        
        logger.info("Creating DB adaptors for test_slice_" + releaseNumber + ", test_slice_" + prevReleaseNumber + "_snapshot and gk_central");
        
-       String sliceUsername = props.getProperty("sliceUsername");
-       String slicePassword = props.getProperty("slicePassword");
-       String sliceHost = props.getProperty("sliceHost");	   
-       String sliceDatabase = props.getProperty("sliceDatabase");
+       String sliceUsername = props.getProperty("release.database.user");
+       String slicePassword = props.getProperty("release.database.password");
+       String sliceHost = props.getProperty("release.database.host");	   
+       String sliceDatabase = props.getProperty("slice_current.name");
        MySQLAdaptor dbaSlice = new MySQLAdaptor(sliceHost, sliceDatabase, sliceUsername, slicePassword, port);
        
-       String prevSliceDatabase = props.getProperty("prevSliceDatabase");
+       String prevSliceDatabase = props.getProperty("slice_previous.name");
        MySQLAdaptor dbaPrevSlice = new MySQLAdaptor(sliceHost, prevSliceDatabase, sliceUsername, slicePassword, port);
        
-       String gkCentralUsername = props.getProperty("gkCentralUsername");
-       String gkCentralPassword = props.getProperty("gkCentralPassword");
-       String gkCentralHost = props.getProperty("gkCentralHost");
-       String gkCentralDatabase = props.getProperty("gkCentralDatabase");
+       String gkCentralUsername = props.getProperty("curator.database.user");
+       String gkCentralPassword = props.getProperty("curator.database.password");
+       String gkCentralHost = props.getProperty("curator.database.host");
+       String gkCentralDatabase = props.getProperty("gk_central.name");
```

Comment:
For consistency, maybe this could be "curator.database.name"?

Reply:
That would work for this particular case, but since release has multiple DBs it wouldn't be as clean. I could do something like 'release.current.database.name' if that seems better

Reply:
For sure, I think it's okay to keep "slice_current.name", "release_current.name", etc. for release databases since there are many of them, but for the curator database it should only be one being used and since it's possible not to be using gk_central, so "curator.database.name" or "curator_database.name" might be better?

Reply:
Fair point, I can make that change

Reply:
Done